database-seed/mongo-loader.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const path = require('path');
+const fs = require('fs');
+const DATABASE_LINK = process.env.DATABASE_LINK || require('../config.js').DATABASE_LINK;
+
+mongoose.connect(DATABASE_LINK);
+
+const productImageScheme = mongoose.Schema({
+    productID: {
+        type: Number,
+        unique: true
+    },
+    images: [{
+        url: String,
+        sort: Number
+    }]
+})
+
+let Product = mongoose.model('ProductImage', productImageScheme);
+
+const seedDatabase = () => {
+    let documents = JSON.parse(retrieveData());
+    documents.forEach((document) => {
+        Product.updateOne({'productID':document.productID}, document, {upsert: true}, (err) => {
```

Comment:
You should drop your database before you seed

Reply:
Done, I have updated a new promisified version.  Please take a look!